src/commands/Tools/poll.ts

```diff
@@ -141,10 +141,10 @@ export default class extends SkyraCommand {
 	public async _resolveUsers(message: KlasaMessage, users: string[]) {
 		const output: string[] = [];
 		for (const user of users) {
-			let resolved: string | undefined;
+			let resolved: string | null | undefined;
```

Comment:
What's the point of `null | undefined` when you're not checking specifically `=== null` or `=== undefined` afterwards?

Reply:
https://github.com/kyranet/Skyra/blob/4cb98c2aa110d0ef4ae2aa062920acf27705635d/src/commands/Tools/poll.ts#L147

Returns `string | null`.